service/framework.go

```diff
@@ -317,7 +331,24 @@ func NewVersionedResources(config Config, k8sClient kubernetes.Interface, awsCon
 		s3BucketObjectConfig.Logger = config.Logger
 		s3BucketObjectConfig.RandomKeyWatcher = keyWatcher
 
-		s3BucketObjectResource, err = s3objectv2.New(s3BucketObjectConfig)
+		s3BucketObjectResourceV1, err = s3objectv2.New(s3BucketObjectConfig)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	var s3BucketObjectResourceV2 framework.Resource
+	{
+		s3BucketObjectConfig := s3objectv2.DefaultConfig()
+		s3BucketObjectConfig.AwsService = awsService
```

Comment:
FYI in other places we started using a simple `c` for the config naming. Since this is scoped it always works and is shorter and thus easier to read/scan.

Reply:
Nice, changed for this resource, I can change the rest of them in a followup.

Reply:
Easy, we can just apply this over time. No need to overdo this now.